airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -187,6 +188,14 @@ const ConnectorServiceTypeControl: React.FC<{
           "9da77001-af33-4bcd-be46-6252bf9342b9", // Shopify
         ]
       : [];
+  if (
+    window.CLOUD === "true" &&
+    workspace.workspaceId === "8fda1978-22bc-466c-a9d5-eaf18bb705a9"
```

Comment:
@sherifnada do you know what workspace we should enable this for?

Reply:
@girarda let's do `54135667-ce73-4820-a93c-29fe1510d348`